### DISCUSSION TITLE
Make Material 3 default in Sketchware Pro applications

### DISCUSSION THREAD

**Discussion by RoyTheReal** [Sketchware-Pro/Sketchware-Pro#1289] [Ideas]
I kindly request that you consider updating the **default styles.xml** file and other relevant configurations to ensure that applications are set to use Material 3 components by default. At present, manually replacing these components is suboptimal, and an issue arises where the toolbar text color remains white, even after switching to Material 3. According to Material 3 design guidelines, the toolbar (or action bar) should display black text. Adjusting the default settings would not only align with these guidelines but also provide a more seamless and consistent implementation experience for us developers that use Sketchware Pro.

**Comment by subhash16** [comment]
Yes Good Idea

**Comment by AraafRoyall** [comment]
very bad idea. material design 3 app requires that library to be enabled which makes even 40KB app to 4MB.

wrost idea. we never need this

**Reply by waelchateur** [reply]
i can sent u 100kb project with 100% md3 that 4mb are unreal my bro

**Comment by htmlistrash** [comment]
It would be better to have an option to enable 100%  Md3

**Reply by RoyTheReal** [reply]
Yes along with dynamic colors

**Comment by TheRealAshik** [comment]
Appreciated 👏

**Comment by laymintu** [comment]
I don't like M3 default

**Comment by RoyTheReal** [comment]
Well just make it an option, it doesn't have to be default.
